bool load_file
---
bool success